### PR TITLE
Remove the paper-button in the README

### PR DESCRIPTION
**Pull request by abdonrd**
I proposed to simplify a bit the README example.

### CODE REVIEW

**Review by rictic — APPROVED** [review]
Sure, succinctness is better here.

File: README.md [hunk 1]
```diff
@@ -50,13 +49,12 @@ Then apply the `LazyImportsMixin` mixin (or `LazyImportsBehavior`) to your eleme
...
 ```html
 <link rel="import" href="../../polymer/polymer.html">
-<link rel="import" href="../../paper-button/paper-button.html">
 <link rel="import" href="../lazy-imports-behavior.html">
 
 <dom-module id="upgrade-button">
   <link rel="lazy-import" group="lazy" href="lazy-element.html">
   <template>
-    <paper-button id="button" on-tap="buttonPressed">Upgrade Element</paper-button>
+    <button on-tap="buttonPressed">Upgrade Element</button>
```
Comment (stramel) [inline README.md:57]: @abdonrd Doesn't `on-tap` need the gesture mixin?

File: README.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ Then apply the `LazyImportsMixin` mixin (or `LazyImportsBehavior`) to your eleme
 ### Polymer 2.0 – LazyImportsMixin
 
 ```html
-<link rel="import" href="../../polymer/polymer-element.html">
-<link rel="import" href="../../paper-button/paper-button.html">
+<link rel="import" href="../../polymer/polymer.html">
```
Comment (stramel) [inline README.md:24]: @abdonrd I think this got accidentally switched back to polymer instead of the element?

Comment (abdonrd) [reply README.md:24]: What? 😮
Somethin wrong with the #33. 😕

File: README.md (hunk 1 above)
Comment (abdonrd) [reply README.md:57]: Oh, right! I think so.

File: README.md (hunk 2 above)
Comment (abdonrd) [reply README.md:24]: https://github.com/Polymer/lazy-imports/pull/35

File: README.md (hunk 1 above)
Comment (abdonrd) [reply README.md:57]: https://github.com/Polymer/lazy-imports/pull/35

**Pull request merged**